src/interface.rs

```diff
@@ -351,7 +364,7 @@ mod tests {
 
         let interface = Interface::Datastream(datastream_interface);
 
-        let deser_interface = Interface::from_str(interface_json).unwrap();
+        let deser_interface = Interface::from_str(interface_json).unwrap(); //allow_panic
```

Comment:
what does it mean this comment?

Reply:
I have a script that checks and warns for all the unwrap and panic, I'll remove the comments if you want